spec/javascripts/app/views/conversations_inbox_view_spec.js

```diff
@@ -0,0 +1,178 @@
+describe("app.views.ConversationsInbox", function() {
+  describe("initialize", function() {
+    beforeEach(function() {
+      spec.loadFixture("conversations_read");
+      $("#conversation-new").removeClass("hidden");
+      $("#conversation-show").addClass("hidden");
+    });
+
+    it("call app.views.ConversationsForm()", function() {
+      spyOn(app.views.ConversationsForm.prototype, "initialize");
+      new app.views.ConversationsInbox();
+      expect(app.views.ConversationsForm.prototype.initialize).toHaveBeenCalled();
+    });
+
+    it("call setupConversation()", function() {
```

Comment:
Nevermind, I fixed it in https://github.com/diaspora/diaspora/pull/7108/commits/8579b974b79894eb2fbdfd65eead522fe9d2155a.